### DISCUSSION TITLE
Remove issue categories from templates?

### DISCUSSION THREAD

**Discussion by SpencerKaiser** [AmericanAirlines/Hangar#272] [General]
Our current template includes `[FEATURE REQUEST] - ` and `[BUG] - ` as part of the title. Isn't this redundant given the `enhancement` and `bug` labels? Maybe we update the template and simple add ~`needs-review`~ `needs-triage` along with one of those other labels... thoughts?

<details>
  <summary>Example (Click me)</summary>
  <img width="240" alt="image" src="https://user-images.githubusercontent.com/6445731/107427597-37bc2980-6ae7-11eb-9f66-fffde1823fe4.png">
</details>

**Comment by johnkahn** [comment]
Sounds good 👍  My only pedantic thing is I'd use the label `needs-triage` though, `needs-review` sounds like it would be for PRs

**Reply by SpencerKaiser** [reply]
Ah, good point, I agree with that entirely.
